<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_aziz</class>
 <widget class="QMainWindow" name="gestion_aziz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>951</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/res/img/hand-painted-watercolor-background-with-sky-clouds-shape_24972-1095.jpg);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Materiel</string>
     </attribute>
     <widget class="QLineEdit" name="rech_materiel">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="table_materiel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>71</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#5500ff;&quot;&gt;Gestion Materiel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="materiel_sel_col">
      <property name="geometry">
       <rect>
        <x>591</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quantite</string>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>70</y>
        <width>221</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Form</string>
      </property>
      <widget class="QComboBox" name="type_2">
       <property name="geometry">
        <rect>
         <x>91</x>
         <y>90</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Arme</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Protection</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accessoire</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chien</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantite</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="quantite">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="id">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>60</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="actionedit_materiel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actionadd_materiel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actiondelete_materiel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Vehicule</string>
     </attribute>
     <widget class="QTableView" name="table_vehicule">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>651</width>
        <height>341</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#5500ff;&quot;&gt;Gestion Vehicule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech_vehicule">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="vehicule_sel_col">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Matricule</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Couleur</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consomation</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>641</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>70</y>
        <width>211</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Form</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Couleur</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="matricule">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="consomation">
       <property name="geometry">
        <rect>
         <x>111</x>
         <y>120</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="couleur">
       <property name="geometry">
        <rect>
         <x>91</x>
         <y>90</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Gris Artense</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bleu Celebes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bleu Bourrasque</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Noir</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QComboBox" name="type">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Patrouille</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Banalise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tout-terrain</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unite canine</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Consomation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actionadd_vehicule">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actionedit_vehicule">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actiondelete_vehicule">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
